database/migrations/2020_02_12_000000_create_posts_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->text('title');
+            $table->text('content');
+            $table->bigInteger('author_id')->unsigned();
```

Comment:
Use user_id instead

Reply:
Really niga?

Reply:
Yes

Reply:
Since your pull request has not yet been admited to master, you can still edit that migration

Reply:
Must I change the getAuthorId method to getUserId?

Reply:
I changed the getAuthorId method to getUserId. And did the required changes.